tests/GenerateScriptTests/CompareDirs.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Xunit.Abstractions;
+
+namespace GenerateScriptTests;
+
+public class CompareDirs
```

Comment:
The class shouldn't be public as it doesn't need to be accessible from the outside.

```suggestion
internal class CompareDirs
```